src/Lykke.Service.Limitations/Settings/AppSettings.cs

```diff
@@ -1,38 +1,26 @@
-﻿using JetBrains.Annotations;
+﻿using System;
+using JetBrains.Annotations;
 using Lykke.Service.Limitations.Core.Domain;
 using Lykke.SettingsReader.Attributes;
 using System.Collections.Generic;
+using Lykke.Sdk.Settings;
 
 namespace Lykke.Service.Limitations.Settings
 {
     [UsedImplicitly(ImplicitUseTargetFlags.WithMembers)]
-    public class AppSettings
+    public class AppSettings : BaseAppSettings
     {
         public LimitationsSettings LimitationsSettings { get; set; }
-
-        public SlackNotificationsSettings SlackNotifications { get; set; }
-
-        public MonitoringServiceClientSettings MonitoringServiceClient { get; set; }
-
+        
         public RateCalculatorServiceClient RateCalculatorServiceClient { get; set; }
-    }
+        public AssetServiceClient AssetsServiceClient { get; set; }
 
-    public class SlackNotificationsSettings
-    {
-        public AzureQueuePublicationSettings AzureQueue { get; set; }
+        public RabbitMqSettings SagasRabbitMq { get; set; }
```

Comment:
Please use the new approved format for such settings.

    public CqrsSettings Cqrs { get; set; }
public class CqrsSettings
{
[AmqpCheck]
public string RabbitConnString { get; set; }
}